events/models.py

```diff
@@ -207,6 +207,10 @@ def get_sign_up_max_participants(self):
             return "Ingen Begränsning"
         return self.sign_up_max_participants
 
+    def exclude_indexing(self):
+        grace_period = timedelta(days=1)  # Adjust this to change the grace period
```

Comment:
Maybe adjust grace period to 7 days, and check that it's not in the old events list